CHANGELOG.md

```diff
@@ -32,6 +38,57 @@ NB: this is not necessary if a user has `READONLY = 2` mode as it allows to modi
 
 See also: [readonly documentation](https://clickhouse.com/docs/en/operations/settings/permissions-for-queries#readonly).
 
+### New features
+
+- Added `url` configuration parameter. It is intended to replace the deprecated `host`, which was already supposed to be passed a valid URL.
+- It is now possible to configure most of the client instance parameters with a URL. The URL format is `http[s]://[username:password@]hostname:port[/database][?param1=value1&param2=value2]`. The name of a particular parameter is supposed to reflect its path in the config options interface. The following parameters are supported:
+
+  - `readonly` - boolean. See below [1].
```

Comment:
optional: what if we use a table representation? 

| name | type |
| - | - | 
| readonly[1] | boolean |
| application_id | string |